streaming/index.js

```diff
@@ -16,9 +17,12 @@ const WebSocket = require('ws');
 
 const environment = process.env.NODE_ENV || 'development';
 
-dotenv.config({
-  path: environment === 'production' ? '.env.production' : '.env',
-});
+// Only attempt to load dotenv outside of docker containers:
+if (!isDocker()) {
+  dotenv.config({
+    path: environment === 'production' ? '../.env.production' : '../.env',
+  });
+}
```

Comment:
I'm not sure why we'd special-case this, and it seems unrelated to the split. Did I miss something?

Reply:
Inside a docker container, the .env file doesn't need to be / shouldn't be losded, as all environment variables should come from the container runtime

Reply:
Yeah I get that, but that seems unrelated to the split, and I don't think the Rails part does that special-casing, so I'd omit this for simplicity and consistency, unless there is a stronger reason.

Reply:
Also dotenv should use the environment variables by default if they are set, I am not sure it is an issue if it uses the defaults in `.env` / `.env.production`?